src/Number.php

```diff
@@ -22,7 +22,7 @@ final protected function __construct(protected BigDecimal $value, protected int
     }
 
     /** @see RoundingMode for available rounding mode constants */
-    public static function of(string|int|float|BigNumber|Number $value, ?int $roundingMode = null): static
+    public static function of(string|int|float|BigNumber|Number $value, ?int $roundingMode = null, ?int $decimalPlaces = 2): static
```

Comment:
This doesn't seem to actually be used? 🤔 Or is this just for future use if necessary?

Reply:
Nah, this was my bad